README.md

```diff
@@ -91,11 +91,22 @@ The following attributes are available for all models of `viamrtsp` discovery se
     {
       "user": "<USERNAME2>",
       "pass": "<PASSWORD2>"
-    },
```

Comment:
While we're here - let's move this to the top and run viam module update.

Reply:
sure, ill get the other fixes for this doc too then